lib/battery-voltage.js

```diff
@@ -16,13 +16,13 @@ function BatteryVoltage(opts) {
 util.inherits(BatteryVoltage, events.EventEmitter);
 
 BatteryVoltage.prototype._onSensorData = function(value) {
-  var voltage = (value * 1.1) / 1000;
+  var voltage = parseFloat(((value * 1.1) / 1000).toFixed(4));
```

Comment:
Can we change this to use Math.round instead? Using strings seems clunky. Something like: `value = Math.round(value * 1000) / 1000;`